examples/vision/siamese.py

```diff
@@ -0,0 +1,232 @@
+
+"""
+# Siamese Network with Contrastive Loss
+
+Author: [Mehdi](https://github.com/s-mrb)<br>
+Date created: 2021/05/02<br>
+Last modified: 2020/05/03<br>
+Description: Similarity Learning Using Siamese Network with Contrastive Loss
```

Comment:
Suggestion:

```suggestion
Description: Similarity learning using the Siamese network with a contrastive loss
```

- Use `Sentence case for titles, descriptions, and section names` in line with most example on https://keras.io/examples/ (including the original ones by @fchollet)